index.js

```diff
@@ -119,8 +119,6 @@ const parseCookie = (url, cookie) => {
 	return returnValue;
 };
 
-const imagesHaveLoaded = () => [...document.images].map(element => element.complete);
```

Comment:
This was returning an array of `true/false` before;  always `truthy` and thus never waited.